### DISCUSSION TITLE
Issue accessing working Journal in PK-Sim during Population Simulations

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1882] [Questions & Problems] [answered]
Hi all, my workflow typically involves:
1. Creating models in PK-Sim.
2. Adjusting some reactions in MoBi.
3. Exporting the adjusted PKML file from MoBi back to PK-Sim for population simulations.

This process has worked fine in the past. However, I'm now facing a problem:
1. I created a working journal in PK-Sim where I added the observed data.
2. Upon attempting to evaluate the population simulation against the observed data, I am unable to access the working journal. The journal does not appear when I try to select or view it in PK-Sim. but strangely, when I open the model in MoBi, the journal is available and opens without issues.
I have used this approach successfully multiple times before, but now PK-Sim seems unable to recognize the working journal, even though it exists.

Has anyone faced a similar issue or have any insights into what might be causing this?

Thanks in advance, 
Aya

**Comment by Yuri05** [comment]
No real idea what happens here...
Can you create a local copy of your working journal and then select this copied journal file in PK-Sim via `File ➡️ Select Journal` ?

**Comment by rwmcintosh** [comment, marked as answer]
Is it that the Journal window is not present, or that it's present, but empty? If it's not present, you could try to reset the UI.

Close all your PK-Sim instances and navigate

C:\Users\\<**USERID**>\AppData\Roaming\Open Systems Pharmacology\PK-Sim\\<**Version**>

substituing USERID and VERSION as appropriate. The start PK-Sim again

**Reply by AyaSaleh90** [reply]
It was not present at all in any PKsim model, so I followed that path and it worked only when I deleted the "UserSettings.xml" file. 
it works fine now, but it also deleted the templates I had earlier.

Thanks a lot !

**Reply by rwmcintosh** [reply]
Oh yeah, I missed that part. Delete the file from the folder. Oops
